templates/main/about/partners.html

```diff
@@ -0,0 +1,119 @@
+{% extends "main/base.html" %}
+{% block title %}Partners{% endblock %}
+{% block og_title %}Partners{% endblock %}
+{% block og_description %}List and Links to Partner servers{% endblock %}
+{% block content %}
+    <div class="uk-section">
+        <div class="uk-container uk-container-small">
+            <article class="uk-article">
+                <h1 class="uk-article-title hover-title" id="top">
+                    Partners
```

Comment:
Can we generate this from a JSON file or something? It seems pretty standard and it'd be nice if we could modify it easily later on

Reply:
The Partners cards are now generated from `static/partners.json`.